.travis.yml

```diff
@@ -5,12 +5,16 @@ os:
 language: go
 
 go:
-    - 1.7
+    - 1.9.x
 
 env:
     - GO15VENDOREXPERIMENT=1
```

Comment:
What is this `GO15VENDOREXPERIMENT=1` and is it still needed?

Reply:
Leftovers from go 1.5 (when go first started experimenting with "vendor" directories"). You can remove it.